README.md

```diff
@@ -2,15 +2,22 @@
 
 # bv-ui-core
 
-This project provides a central location for common Bazaarvoice UI code. It is intended to be installed into a project via npm. Individual modules are authored as CommonJS modules, to be consumed by Webpack or another build tool that can ingest CommonJS.
+This project provides a central location for common Bazaarvoice UI code. It is
+intended to be installed into a project via npm. Individual modules are authored
```

Comment:
@tnunamak can you open a new PR with these changes? This PR has already landed. Thanks :)